test/e2e/scheduling/preemption.go

```diff
@@ -467,7 +467,13 @@ var _ = SIGDescribe("PreemptionExecutionPath", func() {
 		}
 	})
 
-	ginkgo.It("runs ReplicaSets to verify preemption running path", func() {
+	/*
+	  Release : v1.16
+	  Testname: Scheduler: Preemption of ReplicaSet Pods
+	  Description: ReplicaSets MUST be scheduled within a reasonable
+	  amount of time. ReplicaSet pods MUST NOT be over-preempted.
```

Comment:
I don't understand what over-preempted means in this context

I would expect a more detailed description that mentions stuff from the comments like:
- creating a fake resource
- scheduling replicasets to take up 700/800 of that fake resource
- scheduling one more high priority replicaset that takes 400/800 of that resource
- verifying that lowest priority replicaset pods are preempted first

but it's also unclear to me what this is verifying that other preemption tests are not